tuf/api/metadata.py

```diff
@@ -954,8 +961,8 @@ def add_key(self, role: str, key: Key) -> None:
             self.roles[role].keyids.append(key.keyid)
         self.keys[key.keyid] = key
 
-    def remove_key(self, role: str, keyid: str) -> None:
-        """Removes key from ``role`` and updates the key store.
+    def revoke_key(self, role: str, keyid: str) -> None:
```

Comment:
this (and add_key()) are now named exactly like the Targets methods, but have exactly opposite argument order...

Reply:
Yes, and I was wondering should I change them just for the sake of consistency?
We changed the order just because we want to make `role` optional, but don't want to do that in `Root`.
So, what do you think?

Reply:
> Yes, and I was wondering should I change them just for the sake of consistency?

Yes, that seems to be what we agreed on [above](https://github.com/theupdateframework/python-tuf/pull/2010#issuecomment-1149807399).

Reply:
Me earlier:
> I think all *_key() methods should have the same logic (so yes root methods should change argument order and name if the targets methods do, even if they don't practically need to).

I think it's bad if we have methods that look very similar but are not. So IMO either we don't change any argument order or we change all of them.

Reply:
Okay, will make the arguments in the same order.